src/nft/components/collection/Filters.css.ts

```diff
@@ -26,57 +26,67 @@ export const container = style([
 ])
 
 export const rowHover = style([
-  sprinkles({
-    borderRadius: '12',
-  }),
   {
     ':hover': {
-      background: themeVars.colors.backgroundSurface,
+      background: themeVars.colors.backgroundInteractive,
+      borderRadius: 12,
```

Comment:
it seems like everywhere that uses `rowHover` already has `borderRadius: '12'`, so it seems like there is no difference between `rowHover` and `rowHoverOpen`